src/controls/qml/LayerStack.qml

```diff
@@ -56,11 +57,27 @@ Item {
         params["x"] = 0
         params["y"] = 0
         if(typeof firstPage != 'undefined' && firstPage.status === Component.Ready) {
-            var itm=firstPage.createObject(content, params)
+            itm=firstPage.createObject(content, params)
```

Comment:
itm is a pretty bad name, could you use this opportunity to rename it to something like firstPageInstance ?